elasticmagic-query-filters/src/commonMain/kotlin/dev/evo/elasticmagic/qf/QueryFilters.kt

```diff
@@ -77,6 +77,12 @@ class QueryFiltersResult(
         return results[filter.name] as R
     }
 
+    fun <R : FilterResult> get(name: String): R? {
```

Comment:
Either we can make its signature as `inline fun <reified R : FilterResult> get(name: String): R?` or we shouldn't cast inside the method. Making the method `indline` it requires `results` property to be public. But to be honest I wouldn't want to make `results` property public because of its type. So I prefer to drop the cast.